### ISSUE TITLE
Change timeouts to the web.config

### ISSUE THREAD

**Issue by LaurentGoderre** [wet-boew/web-reporting#56]
Please change the following lines:

From this:

```
<add key ="SessionWarning" value ="110"/>
```

To this:

```
<add key ="SessionWarning" value ="50"/>
```

From this

```
<sessionState timeout="120"/>
```

To this:

```
<sessionState timeout="60"/>
```

From this

```
 <forms name=".ASPXFORMSAUTH" timeout="120" loginUrl="index.aspx" protection="All"/>
```

To this:

```
 <forms name=".ASPXFORMSAUTH" timeout="60" loginUrl="index.aspx" protection="All"/>
```

**Comment by LaurentGoderre** [comment]
This should fix #6

**Comment by FarazAliKhan** [comment]
Fixed.
